docker/bench-all.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
-# Copyright (c) 2022, NVIDIA CORPORATION.
+# Copyright (c) 2022-2025, NVIDIA CORPORATION.
 
 set -e
 
 function logger {
-    echo -e "\n$@\n"
+    echo -e "\n${1}\n"
```

Comment:
Here's what `shellcheck` was unhappy about:

```text
In docker/run.sh line 6:
    echo -e "\n$@\n"
               ^-- SC2145 (error): Argument mixes string and array. Use * or separate argument
```

There aren't any uses of this function in this script where more than 1 positional argument is passed, so I think just changing it to `${1}` should be fine.